city_scrapers/spiders/il_sex_offender_management.py

```diff
@@ -0,0 +1,249 @@
+import json
+import re
+from datetime import datetime, timedelta
+from io import BytesIO, StringIO
+
+import scrapy
+from city_scrapers_core.constants import BOARD, CANCELLED, PASSED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from pdfminer.high_level import extract_text_to_fp
+from pdfminer.layout import LAParams
+
+
+class IlSexOffenderManagementSpider(CityScrapersSpider):
+    name = "il_sex_offender_management"
+    agency = "Illinois Sex Offender Management Board"
+    timezone = "America/Chicago"
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+    start_urls = [
+        "https://www2.illinois.gov/idoc/" + "Pages/SexOffenderManagementBoard.aspx"
+    ]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        for link in response.css(
```

Comment:
`.soi-article-content` is a more general selector that's less likely to change and still seems to pull the same content, does that get at what you're looking for here?